### PR TITLE
added aries symbol

### PR DESCRIPTION
**Pull request by Yugal41735**
Please check and tell me if there are any errors

### CODE REVIEW

**Comment by Yugal41735** [comment]
And if u want me add something else do let me know

**Comment by DenverCoder1** [comment]
Thanks for the enthusiasm to contribute.  <sub>You're the first person today to get it right on the first try.</sub>

Unfortunately, adityagi02 has opened a PR for it, how about try something new such as one of these:

```
\texttrademark	™	
\copyright	©	
\circledR	®	
```

**Comment by Yugal41735** [comment]
ok no problem

**Comment by Yugal41735** [comment]
You can check and tell me if it is working fine

File: dictionary.txt [hunk 1]
```diff
@@ -866,7 +866,10 @@ _j	ⱼ
 \lrblacktriangle	◢	
 \llblacktriangle	◣	
 \ulblacktriangle	◤	
-\urblacktriangle	◥	
+\urblacktriangle	◥
```
Comment (DenverCoder1) [inline dictionary.txt:869]: Looks good, there's a tab missing at the end of this line, though

```suggestion
\urblacktriangle	◥	
```

Comment (Yugal41735) [reply dictionary.txt]: Done,

**Review by DenverCoder1 — APPROVED** [review]

**Comment by DenverCoder1** [comment]
Looks great, thanks for contributing!

**Pull request merged**